doc/Extensions/Applications.md

```diff
@@ -769,6 +770,24 @@ It is worth noting that this only monitors a single pool. If you want to monitor
 
 The application should be auto-discovered as described at the top of the page. If it is not, please follow the steps set out under `SNMP Extend` heading top of page.
 
+### PHP-OPCACHE
+#### SNMP Extend
+1. Copy the PHP script, opcache.php, to the desired host (needs to be surved true your website).
+In this example the www root of LibreNMS is used, change as disire
+`wget https://github.com/librenms/librenms-agent/raw/master/snmp/opcache.php -O /opt/librenms/html/`
```

Comment:
You need to rebase for this file anyway but you shouldn't save the file there, `/etc/snmp/` is the location used elsewhere.

Reply:
@laf its a php file
Opcache needs to be prosessed true PHP-FPM u cant do it via CLI
So i dont whant to make an directory inside /etc/snmp/ for securety reasons

Reply:
(well u can but u need extra depancies to do it)